src/main/modules/oidc/index.ts

```diff
@@ -116,6 +119,7 @@ export class OidcMiddleware {
     } catch (e) {
       if (e instanceof InProgressDivorceCase) {
         const token = encodeURIComponent(req.session.user.accessToken);
+        logger.info(`Case ID ${req.session.userCase.id} being redirected to old divorce`);
```

Comment:
they might not have an existing NFD case at this point, so need to use 
req.session.userCase?.id